lessonmacgyver.tex

```diff
@@ -0,0 +1,117 @@
+      \item (Advanced) Potential vs kinetic energy
+      \item (Cooperative) How to work on different pieces of the same system
+      and tie them together in the end
+    \end{itemize}
+
+    \subsection{Key Vocabulary Terms}
+    \begin{itemize}
+      \item Engineering
+      \item (Intermediate) Conservation of energy, energy
+      \item (Advanced) Potential, kinetic energy
+    \end{itemize}
+
+    \subsection{Materials}
+      \begin{itemize}
+        \item Cardboard box
+		\item String
+		\item Tape
+		\item Netting
+		\item Erasers
+		\item Balls (marbles, rubber balls, etc)
+		\item Pencils
+		\item Cereal boxes
+		\item Wedge
+		\item Cardboard tube
+		\item Waterbottle
+		\item And anything else that would make this lesson fun!
+      \end{itemize}
+
+  \section{Procedure}
+    \subsection{Setup (10 minutes)}
+	\paragraph{}
+      Place the lego crab on the ground, and tape a square around it. Box should be large enough so that crab is fully enclosed, and has a few inches
+	of space around it. 
+	Place materials in piles around the crab.
+
+    \subsection{Introduction (5 minutes)}
+      Introduce using a video of a cool Rube Goldberg Machine (we suggest
+      \href{https://www.youtube.com/watch?v=qybUFnY7Y8w}). This is a fun
+      music video, so if you can set up sound, it's all the better.
+
+      Now introduce the scenario. For example:
+      "After 3 days stuck at sea, you castaways land on a deserted island.
+      All that's left on the wrecked boat is a fishing line (hold up the string),
+      an empty water bottle (which they can fill), a cardboard box (chest from
+      their pirate treasure), some erasers, a wedge and a board,
+      a cardboard tube, a cereal box, and a ball.''
+
+      Explain to the kids that getting too close to the crab spooks it.
+      They must make a machine to capture it without standing next to it.
+
+    \subsection{Demonstration}
+      Show the ball hit the erasers, so they fall over like dominoes.
+
+    \subsection{Guided Practice}
+		\begin{itemize}
+			\item Show that the box can serve as a trapping mechanism to capture the crab
+			\item If we have a net, show that we can trap the crab in the net
+			\item Show that a tube can be used to roll a ball into domino-erasers, which can hit a wedge, tipping 
+					a waterbottle and knocking into a cereal box ...
+			\item Basically, show different combinations of how to use the materials
+			\item Use tape at some point
+			\item Point out where energy transfer happens
+		\end{itemize}
+
+      \subsubsection{sticking points:}
+      \begin{itemize}
+        \item A box can be held up with a ruler or a string - there's more than one way to capture the crab with the box
+	    \item Encourage them to compare and contrast ideas, rather than choosing one and sticking with it
+		\item Encourage students to keep trying, even if their ideas fail at first
+		\item Point out places in their own designs where energy is transferred
```

Comment:
Spacing is real weird, I'll get CI to check this as some point soon but for now we should make sure the files are at least consistent within one file